activerecord/lib/active_record/migration.rb

```diff
@@ -736,13 +718,16 @@ def disable_ddl_transaction!
         @disable_ddl_transaction = true
       end
 
+      # Raises ActiveRecord::PendingMigrationError error if any migrations are pending
+      # for all database configurations in an environment.
       def check_pending_migrations # :nodoc:
```

Comment:
You need to remove the nodoc, otherwise the above docs won't show in the API docs.

Reply:
Removed.